pkg/upgradeManager/upgradeManager.go

```diff
@@ -0,0 +1,43 @@
+package upgrademanager
+
+import (
+	"os"
+	"os/exec"
+)
+
+type UpgradeManager struct {
+}
+
+func NewUpgradeManager() *UpgradeManager {
+	return &UpgradeManager{}
+}
+
+func (m *UpgradeManager) CheckIfDatreeInstalledUsingBrew() bool {
+	_, err := exec.Command("brew", "list", "datree").Output()
+	return err == nil
+}
+
+func (m *UpgradeManager) Upgrade() error {
+
+	shellScript := exec.Command("curl", "https://get.datree.io")
```

Comment:
This seems to work as well, replacing the entire function with 3 lines of code.

```
oneLineInstallationCommand := exec.Command("bash", "-c", "curl https://get.datree.io | /bin/bash")
oneLineInstallationCommand.Stdout = os.Stdout
return oneLineInstallationCommand.Run()
```

Is there any benefit to using the current 20 lines of code instead of the suggested 3? I'm asking out of curiosity

Reply:
I have barely seen `-c` option and if you're confident about the one-liner. I would love to see it happening. I have changed the code with the given code block. 👍🏻